org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/messages.properties

```diff
@@ -55,7 +55,7 @@ actionUnbind=Unbind
 actionUpdate=Update All Project Bindings
 actionChangeBinding=Change Binding...
 actionEdit=Edit Connection...
-actionNewServer=New Server Connection...
+actionNewServer=New Connection...
 
 # Servers action set
 actionSetNewServer=Create Server
```

Comment:
Shouldn't this one be updated too?

Reply:
This `actionSetNewServer` message is used in `NewConnectionWizardAction()` with a `setText()` but the text is immediately replaced with another setText() in `BindingsViewActionProvider`. I think we can safely delete this entry